Velaptor/OpenGL/Buffers/TextureGPUBuffer.cs

```diff
@@ -41,9 +43,19 @@ internal sealed class TextureGPUBuffer : GPUBufferBase<TextureBatchItem>
         IGLInvoker gl,
         IOpenGLService openGLService,
         IReactable<GLInitData> glInitReactable,
+        IReactable<BatchSizeData> batchSizeReactable,
         IReactable<ShutDownData> shutDownReactable)
         : base(gl, openGLService, glInitReactable, shutDownReactable)
     {
+        // TODO: Test for null batch reactable and subscription
```

Comment:
Do you need this TODO?
```suggestion
        // TODO: Test for null batch reactable and subscription
```